package.json

```diff
@@ -24,14 +25,23 @@
     "mongoose": "^5.2.2",
     "mongoose-unique-validator": "^2.0.1",
     "morgan": "^1.10.0",
+    "mysql2": "^2.2.5",
+    "nodemon": "^2.0.6",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
+    "pg": "^8.5.0",
     "request": "^2.87.0",
+    "sequelize": "^6.3.5",
     "underscore": "^1.9.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.12.1",
+    "@babel/core": "^7.12.3",
+    "@babel/node": "^7.12.6",
+    "@babel/preset-env": "^7.12.1",
```

Comment:
These dependencies should not be in `devDepedencies` as they will not run in a production environment.